src/components/Home.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux';
+import restaurantsData from '../restaurant.data';
+import './styling/Home.css';
+
+import Card from 'react-bootstrap/Card'
+import Badge from "react-bootstrap/Badge";
+import Image from "react-bootstrap/Image";
+import StarRatings from 'react-star-ratings';
+
+class Home extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      restaurants: restaurantsData,
+    }
+  }
+
+  render() {
+    return (
+      <div>
+        <h1>
+          <Badge variant="secondary">All Restaurants</Badge>
+        </h1>
+        <div className='preview'>
+          {
+            this.state.restaurants.map(({name, imageUrl, rating}) => (
+              <Card style={{ width: '23rem', margin: '3px' }}>
+                <Card.Header>
+                  <Image
+                    src={imageUrl}
+                    style={{
+                      display: "block",
+                      margin: "auto",
+                      width: "158px",
```

Comment:
We should avoid the use of px since things might end up looking different depending on screen resolution

Reply:
good point, will change this